src/context.rs

```diff
@@ -813,6 +828,32 @@ impl ConcreteBaseAudioContext {
         self.connect(&AudioNodeId(LISTENER_NODE_ID), panner, 7, 8);
         self.connect(&AudioNodeId(LISTENER_NODE_ID), panner, 8, 9);
     }
+
+    /// Add the [`AudioListener`] to the audio graph (if not already)
+    pub(crate) fn release_audio_listener(&self) {
```

Comment:
same, I find `release` to be counter intuitive, `register`, `attach`, `connect` ?